client/src/components/PageLayout/Header.tsx

```diff
@@ -16,6 +18,66 @@ const useStyles = makeStyles({
 
 const Header: React.FC<{ classes: Record<string, any> }> = ({ classes }) => {
   const styles = useStyles();
+  const [isAuth, setIsAuth] = useState(false);
+
+  useEffect(() => {
+    const runGoogleAuth = async (script: HTMLElement) => {
+      const response = await fetch(
+        `${process.env.NEXT_PUBLIC_CHAPTER_SERVER}/auth/is-authed`,
+        {
+          method: 'get',
+          credentials: 'include',
+        },
+      );
+      const responseBody = await response.json();
+      const { isAuth } = responseBody;
+
+      if (!isAuth) {
```

Comment:
`isAuth` will always be `false` in this hook because it's initialized to be false, and the hook only runs once on mount.

Reply:
This `isAuth` is refering to the api call in the func.